### ISSUE TITLE
Automation - Count number of issues detected by Accessibility check and update slack message with that value

### ISSUE THREAD

**Issue by IsraelleHub** [department-of-veterans-affairs/va-mobile-app#8524]
# Automation Ticket

## Type of Change
- [x] Update existing automation
- [x] Add new automation functionality
- [ ] Fix automation bug

## What systems/scripts are changing?
- [x] Git Hub Actions
- [ ] CircleCI
- [ ] Fastlane
- [ ] Bash Scripts
- [ ] Issues Templates

## What is the work being done?
Update [documentation_accessibility-checks.yml](https://github.com/department-of-veterans-affairs/va-mobile-app/tree/develop/.github/workflows) workflow to make it count number of issues detected during accessibility checks scanning. this issue is linked to [#8217](#8217: Add accessibility checks into CI for the documentation site) ticket
## Acceptability Criteria

- [ ] Check on va-mobile-build-alert chanel DSVA to see if slack message generate number of issues detected during  scanning

**Comment by IsraelleHub** [comment]
Update: Still trying to figure out a way to have this workflow count number of accessibility issue on each url and totalize it. I was able to have the worflow count the number of url with axe issue without any problem. However, when trying to make to have the workflow count number of url instead, it just timeout out and exiting without completing. [https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/9950165478/workflow](url)
- tried, to increased axe cli timeout. [https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/9950099303/workflow](url)
-  I tried to change some of the setting for the headless chrome browser [https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/9996848453/job/27632081137](url)

**Comment by bischoffa** [comment]
@IsraelleHub can you update the ticket to include the current sprint?

**Comment by IsraelleHub** [comment]
**Reason for Block:** This task, which involves counting the number of accessibility issues for each URL, has encountered persistent timeout errors that have proven challenging to resolve. Due to the time already invested without significant progress, I've decided to deprioritize this task temporarily. I will revisit it periodically, but for now, I am shifting focus to higher-priority tasks.

**Comment by lexicalninja** [comment]
@IsraelleHub is there a branch you were testing this on that i could look at and maybe take a shot at?

**Comment by IsraelleHub** [comment]
@lexicalninja this is the4 branch chanel-8524-automation-accessibility-checks-number-issues and this is the [PR](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/8939). Thank you for looking into it!

**Comment by lexicalninja** [comment]
thank you!

**Comment by lexicalninja** [comment]
@IsraelleHub I was able to figure this one out. On the branch you added an `axe-scan.js` file that i don't think is being used. can i delete that or did you have other plans for that?

**Comment by IsraelleHub** [comment]
@lexicalninja we can delete it. I created it when I was troubleshooting the timeout issues. Thank you
